react/src/components/tooltip/SprkTooltip.js

```diff
@@ -224,6 +235,10 @@ SprkTooltip.propTypes = {
    * the trigger element. Intended for an outside library to capture data.
    */
   analyticsString: PropTypes.string,
+  /**
+   * Vertical alignment of tooltip.
```

Comment:
Some thing for the comment here as well." whats the default setting for the tooltip? As a user I could see it being useful to have that here in the comment. Like something about how this is to override the default setting." Capitalize tooltip too please!

Reply:
https://github.com/sparkdesignsystem/spark-design-system/pull/3976#discussion_r626693432

also added that into the description!